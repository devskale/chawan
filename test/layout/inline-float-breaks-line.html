<!DOCTYPE html>
<div style="display: flow-root">
test1
<div style="float: left; width: 100%">
test2
</div>
<div style="float: left">
test3
</div>
test4
</div>
<div style="display: flow-root">
test1
<div style="float: left; width: 50%">
test2
</div>
<div style="float: left">
test3
</div>
test4
</div>
