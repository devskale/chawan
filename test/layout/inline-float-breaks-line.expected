test1 test4
test2
test3
test2                                   test3test1 test4
